File: src/chameleon.rb
S="DATA";L=S.split("\t");I={{{chameleon:rand(LANGCOUNT)*FIELDCOUNT}}}{{{ouroboros:ENTRYINDEX}}};print L[I]+S.gsub(/[\t\n"\\]/,"\t"=>'\t',"\n"=>'\n','"'=>'\"','\\'=>'\\\\')+L[I+1]

File: src/chameleon.pl
my $S=q:b"DATA";my @L=$S.split("\t");my $I={{{chameleon:LANGCOUNT.rand.Int*FIELDCOUNT}}}{{{ouroboros:ENTRYINDEX}}};print @L[$I]~$S.trans(["\t","\n",'"',"\\"]=>['\t','\n','\\"',"\\\\"])~@L[$I+1];
